src/components/client-add/ClientAdd.jsx

```diff
@@ -0,0 +1,170 @@
+                ${validity.client === false && styles.invalidInput}`}
+              maxLength="30"
+              minLength="2"
+              name="client"
+              onBlur={handleBlur}
+              onChange={({ target }) => setClient(target.value)}
+              required
+              value={client}
+            />
+          </label>
+          <label className={`${styles.label}`} htmlFor="branch">
+            <span className={styles.labelText}>Branch</span>
+            <input
+              className={`${styles.input} ${validity.branch === false &&
+                styles.invalidInput}`}
+              maxLength="30"
+              minLength="2"
+              name="branch"
+              onBlur={handleBlur}
+              onChange={({ target }) => setBranch(target.value)}
+              required
+              value={branch}
+            />
+          </label>
+        </div>
+        <span className={styles.verticalLine} />
+        <div className={styles.inputContainer}>
+          <label className={`${styles.label}`} htmlFor="branchAdress">
+            <span className={styles.labelText}>Branch adress</span>
+            <input
+              className={`${styles.input} 
+              ${validity.branchAdress === false && styles.invalidInput}`}
+              maxLength="30"
+              minLength="2"
+              name="branchAdress"
+              onBlur={handleBlur}
+              onChange={({ target }) => setBranchAdress(target.value)}
+              required
+              value={branchAdress}
+            />
+          </label>
+          <label
+            className={`${styles.label} ${styles.labelHalfWidth}`}
+            htmlFor="postalCode"
+          >
+            <span className={styles.labelText}>Postal code</span>
+            <input
+              className={`${styles.input} 
+              ${validity.postalCode === false && styles.invalidInput}`}
+              maxLength="30"
+              minLength="2"
+              name="postalCode"
+              onBlur={handleBlur}
+              onChange={({ target }) => setPostalCode(target.value)}
+              required
+              value={postalCode}
+            />
+          </label>
+          <label
+            className={`${styles.label} ${styles.labelHalfWidth}`}
+            htmlFor="locality"
+          >
+            <span className={styles.labelText}>Location</span>
+            <input
+              className={`${styles.input} 
+              ${validity.locality === false && styles.invalidInput}`}
+              maxLength="30"
+              minLength="2"
+              name="locality"
+              onBlur={handleBlur}
+              onChange={({ target }) => setLocality(target.value)}
+              required
+              value={locality}
+            />
+          </label>
+        </div>
+      </form>
+    </div>
+  );
+};
+
+ClientAdd.propTypes = {
+  addFormData: PropTypes.func.isRequired,
+  handleFormVisible: PropTypes.func.isRequired
```

Comment:
Consider names to be phrases: "handle form visible" would be incorrect then. Something like "handle form visibility" would be better. Also, `handle` suits events better. When toggling the visibility I'd name it `toggleFormVisibility`.